doc/user/data/self_managed/authentication_setting.yml

```diff
@@ -3,17 +3,28 @@ columns:
   - column: "Description"
 
 rows:
-- "authenticatorKind Value": "**None**"
-  "Description": |
-    Disables authentication. All users are trusted based on their claimed
-    identity **without** any verification. **Default**
-- "authenticatorKind Value": "**Password**"
-  "Description": |
-    Enables [password authentication](#configuring-password-authentication) for
-    users. When enabled, users must authenticate with their password.
+  - "authenticatorKind Value": "**None**"
+    "Description": |
+      Disables authentication. All users are trusted based on their claimed
+      identity **without** any verification. **Default**
+  - "authenticatorKind Value": "**Password**"
+    "Description": |
+      Enables [password authentication](#configuring-password-authentication) for
+      users. When enabled, users must authenticate with their password.
 
-    {{< tip >}}
-    When enabled, you must also set the `mz_system` user password in
-    `external_login_password_mz_system`. See [Configuring password
-    authentication](#configuring-password-authentication) for details.
-    {{</ tip >}}
+      {{< tip >}}
+      When enabled, you must also set the `mz_system` user password in
+      `external_login_password_mz_system`. See [Configuring password
+      authentication](#configuring-password-authentication) for details.
+      {{</ tip >}}
+  - "authenticatorKind Value": "**Sasl**"
+    "Description": |
+      Enables [SASL/SCRAM-SHA-256 authentication](#configuring-saslscram-authentication) for
+      PostgreSQL wire protocol connections. This is a challenge-response authentication
```

Comment:
Could we bold the **PostgreSQL wire protocol connections** ? Even bolded, some people will miss it, but I think it'll help reduce the number of people who'll miss it.